frontend/src/views/Administration.vue

```diff
@@ -201,16 +378,67 @@ export default {
       return this.cfg.sla || {}
     },
     slaDescriptionCompiledMarkdown () {
-      return compileMarkdown(this.sla.description)
+      return compileMarkdown(this.sla.description, this.color)
     },
     slaTitle () {
       return this.sla.title
+    },
+    textColor () {
+      return textColor(this.color)
     }
   },
   methods: {
     ...mapActions([
+      'patchProject',
       'deleteProject'
     ]),
+    onEditTechnicalContact () {
+      this.editTechnicalContact = !this.editTechnicalContact
+      if (this.editTechnicalContact) {
+        this.$nextTick(() => this.$refs.technicalContact.activateMenu())
+      }
+    },
+    updateTechnicalContact (value) {
+      return this.updateProperty('owner', value)
+    },
+    onMenuActive (value) {
+      console.log('onMenuActive', value)
+    },
+    updateDescription (value) {
+      return this.updateProperty('description', value)
+    },
+    updatePurpose (value) {
+      return this.updateProperty('purpose', value)
+    },
+    updateCostObject (value) {
+      if (this.costObjectSettingEnabled) {
+        return this.updateProperty('costObject', value)
+      }
+    },
+    async updateProperty (key, value = null) {
+      const { metadata: { name, namespace } } = this.project
+      try {
+        const mergePatchDocument = {
+          metadata: { name, namespace }
+        }
+        switch (key) {
+          case 'costObject':
+            set(mergePatchDocument, ['metadata', 'annotations', 'billing.gardener.cloud/costObject'], value)
+            break
+          default:
+            set(mergePatchDocument, ['data', key], value)
+            break
+        }
+        // eslint-disable-next-line no-unused-vars
+        const body = await this.patchProject(mergePatchDocument)
+      } catch (err) {
+        const error = new Error(`Failed to update project ${key}`)
```

Comment:
The error message reads too technical, e.g. `Failed to update project costObject`.

Reply:
![image](https://user-images.githubusercontent.com/1574023/81303551-2045a100-907c-11ea-964d-3dde5d2f1df9.png)

Reply:
✔️